ts-apps/jl4-web/src/style.css

```diff
@@ -61,3 +61,16 @@
     border: 1px solid grey;
   }
 }
+
+.quick-input-widget {
```

Comment:
These styles seem quite specific to the monaco editor; for that reason, it'd be better to put them in the `<style> ... </style>` block in `jl4-web/src/routes/+page.svelte`.